app/frontend/src/components/QuestionInput/QuestionInput.tsx

```diff
@@ -65,7 +65,7 @@ export const QuestionInput = ({ onSend, disabled, placeholder, clearOnSend, init
     };
 
     const disableRequiredAccessControl = requireLogin && !loggedIn;
-    const sendQuestionDisabled = disabled || !question.trim() || requireLogin;
+    const sendQuestionDisabled = disabled || !question.trim() || disableRequiredAccessControl;
```

Comment:
I added "disableRequiredAccessControl" to this boolean condition, to align the textfield and button disabled state.